extensions/positron-python/python_files/positron/pinned-test-requirements.txt

```diff
@@ -8,6 +8,7 @@ hvplot==0.8.0 ; python_version < '3.9'
 ibis-framework[duckdb]==9.5.0; python_version >= '3.10'
 ipykernel==6.29.5
 ipywidgets==8.1.5
+lightning==2.5.0.post0
```

Comment:
we do some tests on Python 3.8 still, and it looks like the last version where `lightning` supported 3.8 is `2.3.2`

Reply:
Ah good point. I've updated to support Python 3.8